locales/ru-RU.json

```diff
@@ -159,7 +164,24 @@
   "language": {
     "search": "Поиск"
   },
+  "list": {
+    "add_account": "Добавить аккаунт в список",
+    "cancel_edit": "Отменить редактирование",
+    "clear_error": "Ошибка очистки",
+    "create": "Создать",
+    "delete": "Удалить список",
+    "delete_error": "Произошла ошибка при удалении списка",
+    "edit": "Редактировать список",
+    "edit_error": "Произошла ошибка при изменении списка",
+    "error": "Произошла ошибка при создании списка",
+    "error_prefix": "Ошибка: ",
+    "list_title_placeholder": "Название списка",
+    "modify_account": "Изменить список с помощью аккаунта",
```

Comment:
Where is this actually used? The translation seems suspicious 🤔

Reply:
I don't understand "modify_account"'s purpose. I haven't found where is it.